Header header
int32 count
SegmentedBottle[] bottles
